tools/pot/tests/test_statistics_collector.py

```diff
@@ -54,11 +57,20 @@ def test_statistics_collector_subsets(tmp_path, models, model_name, model_framew
     out = {'MinMaxQuantization': collector.get_statistics_for_algorithm('MinMaxQuantization'),
            'BiasCorrection': collector.get_statistics_for_algorithm('BiasCorrection')}
 
-    refs_file = Path(__file__).parent / 'data/test_cases_refs/statistics_data.txt'
+    refs_file = Path(__file__).parent / 'data/test_cases_refs/statistics_data.json'
+    local_path = os.path.join(tmp_path, '{}_{}.json'.format(model_name, 'statistics_data'))
+    local_file = open(local_path, 'w')
+
     with open(refs_file.as_posix()) as file:
-        refs = json.loads(json.load(file))
+        refs = json.load(file)
 
     eps = 1e-3
+    local_out = copy(out)
+    for algo_name, algo_val in local_out.items():
+        for node_name, node_val in algo_val.items():
+            for stats_name, stats_val in node_val.items():
+                local_out[algo_name][node_name][stats_name] = [v.tolist() for v in stats_val]
+    json.dump(local_out, local_file)
```

Comment:
Is it necessary to save the reference?

Reply:
This is only for further reference update because layer names were changed for this model. We can't guarantee that it will not happen again in the future.